docs/entries/lead.md

```diff
@@ -0,0 +1,56 @@
+# Lead
+
+_Lead_ is a term used in Frappe Books for any potential customer or client who has shown interest in the products or services offered.
+
+## Lead Status
+
+A Lead can have various statuses based on the stage of engagement:
+
+|   # | Status         | Description                                                      |
+| --: | :------------- | :--------------------------------------------------------------- |
+|   1 | Open           | Newly created and yet to be engaged.                     |
+|   2 | Replied        | Replied to initial engagement.                          |
+|   3 | Interested     | Shown interest in the product or service.               |
+|   4 | Opportunity    | Opportunity for business.                  |
+|   5 | Converted      | Converted to a customer or client.                 |
```

Comment:
replace `customer` to `Customer`.

Frappe Books specific words should start with caps